experimental/DoubleAndHyperComplexes/test/linear_strands.jl

```diff
@@ -0,0 +1,63 @@
+@testset "linear strands of complexes of graded modules" begin
+  S, (x, y, z) = graded_polynomial_ring(QQ, [:x, :y, :z])
+
+  S1 = graded_free_module(S, [0])
+  I, inc = sub(S1, [f*S1[1] for f in gens(S)])
+  M = cokernel(inc)
+
+  res, aug = free_resolution(Oscar.SimpleFreeResolution, M)
+
+  str, inc_str = Oscar.linear_strand(res)
```

Comment:
Here you get the linear strand together with its inclusion morphism. There is an optional keyword argument `degree` to specify a different initial degree.